src/osx/core/bitmap.cpp

```diff
@@ -134,6 +134,66 @@ private :
     wxDC*         m_selectedInto;
 };
 
+typedef wxObjectDataPtr<wxBitmapRepresentationData> wxBitmapRepresentationDataPtr;
+WX_DEFINE_TYPEARRAY( wxObjectDataPtr<wxBitmapRepresentationData>, wxBitmapRepresentationArray, dummy );
+
+class WXDLLEXPORT wxBitmapRefData : public wxGDIRefData
+{
+public:
+    wxBitmapRefData()
+    {
+        
+    }
+    
+    wxBitmapRefData(const wxBitmapRefData &tocopy) :
+        m_representations( tocopy.m_representations)
+    {
+        
+    }
+
+    wxBitmapRefData(wxBitmapRepresentationData *rep)
+    {
+        AddRepresentation(rep);
+    }
+    
+    void AddRepresentation(wxBitmapRepresentationData *rep)
+    {
+        m_representations.push_back(wxBitmapRepresentationDataPtr(rep));
+    }
+
+    void AddRepresentations(const wxBitmap& other)
+    {
+        const wxBitmapRefData* otherData = other.GetBitmapData() ;
+        for ( size_t i = 0 ; i < otherData->m_representations.size() ; ++i )
+        {
+            m_representations.push_back(otherData->m_representations.at(i));
+        }
```

Comment:
This should be just `m_reps.insert(m_reps.end(), otherReps.begin(), otherReps.end())`

Reply:
but this is not defined with wxVector, isn't it ?

Reply:
Yes, an oversight :-( I think we should add it to `wxVector<>` too, it's simple enough...